control_protocol.md

```diff
@@ -0,0 +1,293 @@
+# Control protocol
+
+The control protocol transmits messages via its {ref}`transport-layer` from one Component to another one.
+The {ref]}`message-layer` is the common language to understand commands, thus creating a remote procedure call.
+
+
+(transport-layer)=
```

Comment:
These labels seem superfluous? Myst Parser should already autocreate the appropriate anchors (see e.g. the usage in the glossary), here even with the same anchor text.

Reply:
Does it replace spaces with colons?

Reply:
toLowercase, spaces to dashes, remove punctuation and other stuff like parens, slashes. Most complex example is https://github.com/pymeasure/leco-protocol/blame/8a2482956554d1bac07f30521e37d60c2df41da1/glossary.md#L40

Reply:
Docs for how these "slugs" are arrived at here: https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#anchor-slug-structure

Reply:
Do you want to remove that label? Or what was your conclusion?